dask_sql/context.py

```diff
@@ -752,12 +752,11 @@ def _prepare_schemas(self):
                 sql_parameters = function_description.parameters
                 if function_description.aggregation:
                     logger.debug(f"Adding function '{name}' to schema as aggregation.")
-                    # TODO: Not yet implemented
-                    # rust_schema.add_or_overload_aggregation(
-                    #     name,
-                    #     [param[1].getDataType() for param in sql_parameters],
-                    #     sql_return_type.getDataType(),
-                    # )
+                    rust_schema.add_or_overload_aggregation(
+                        name,
+                        [param[1].getDataType() for param in sql_parameters],
+                        sql_return_type.getDataType(),
+                    )
```

Comment:
I realize that I'm the one who left the suggestion to implement `add_or_overload_aggregation` 😅 but since at this point we've validated that there should be no UDF/UDAFs sharing the same name, would it make sense to consolidate this logic into a single function that also takes an `aggregation` boolean?

Reply:
Yeah that sounds good to me! I'll make those changes now.